tests/integration/test_headers_and_subtotals.py

```diff
@@ -374,6 +375,247 @@ def test_subtotals_proportions_2d_cube_with_hs_on_two_dim_by_cell(self):
         ]
         np.testing.assert_almost_equal(slice_.table_proportions, expected)
 
+    def test_subtotal_diffs_1d(self):
+        strand = Cube(CR.CAT_SUBDIFF).partitions[0]
+        counts = strand.counts
+        np.testing.assert_array_equal(counts, (81, 211, 212, 131, 79, 51))
```

Comment:
The issue with this test is that I can't immediately see _what_ is it that I'm supposed to be seeing :) So, if it's the insertion 211 - 79 - 51, that should be stated in the test explicitly. Same for all the other expectations. Also, if the numbers are round, it's better to use the `assert counts == [x, y, z]` than `np.testing.assert_array_equal`, because it's the go-to way of the crunchdb team, and we need to fix all the cube tests to use it as well.

Reply:
An even better approach would be to pass the transformations explicitly, when creating the cube. But I've forgotten if that's possible for insertions?